app/lib/authz-middleware.js

```diff
@@ -7,11 +7,18 @@ const userRoles = {
   admin: 'admin',
   editor: 'editor',
   visitor: 'visitor',
+  teamLead: 'teamlead',
```

Comment:
This is just a test!

```suggestion
  teamLead: 'team lead',
```